.rubocop.yml

```diff
@@ -0,0 +1,138 @@
+AllCops:
+  # Exclude anything that isn't really part of our code.
+  # rails_helper is excluded because it's full of solecisms, but it's mostly
+  # generated code and copy-and-pasted snipets from READMEs.
+  Exclude:
+    - 'Guardfile'
+    - 'vendor/**/*'
+    - 'db/**/*'
+    - 'bin/**/*'
+    - '**/*.gemspec'
+    - '**/Rakefile'
+    - '**/Vagrantfile'
+    - 'spec/rails_helper.rb'
```

Comment:
I wouldn't exculde rails_helper.rb as it'll hold configuration but it'll still be code.